src/daemon/daemon.cpp

```diff
@@ -1849,8 +1848,20 @@ void mp::Daemon::create_vm(const CreateRequest* request, grpc::ServerWriter<Crea
                                                       create_error.SerializeAsString()));
     }
 
+    if (preparing_instances.find(name) != preparing_instances.end())
+    {
+        CreateError create_error;
+        create_error.add_error_codes(CreateError::INSTANCE_EXISTS);
+
+        return status_promise->set_value(grpc::Status(grpc::StatusCode::INVALID_ARGUMENT,
+                                                      fmt::format("instance \"{}\" is being prepared", name),
```

Comment:
"already exists" - IMO clearer, and consistent with if VM was already ready

Reply:
@gerboland, to me, "already exists" implies that if I do `list` or `info foo`, then it will show up, but it does not since the instance doesn't really exist yet.  This is why I chose the "being prepared" wording to differentiate the state to the user in a meaningful way.

Reply:
I will also say I'm open to better wording there, but I don't think "already exists" is the right wording.

Reply:
Suggestions:
- "instance name ... is reserved/locked/unavailable"
- "instance name ... has already been requested"

Reply:
@townsend2010 I understand your motivation, but I see no reason why a user would be interested in such a distinction. IMO the fact that a being-downloaded VM doesn't appear in `list` is a bug.

Reply:
@gerboland, we can agree to disagree on this:)

As a data point, lxd/lxc behaves exactly like this in that a container that is launched and is downloading the image will not show up in `lxc list` until the container's download & prepare is complete.

Reply:
But I do like @ricab's suggestion of putting something like "...has already been requested".  I think I will use that instead.